model/class-a-end-device-lorawan-mac.cc

```diff
@@ -17,6 +17,8 @@
  *
  * Author: Davide Magrin <magrinda@dei.unipd.it>
  *         Martina Capuzzo <capuzzom@dei.unipd.it>
+ *
+ * Modified by: Peggy Anderson <peggy.anderson@usask.ca>
```

Comment:
I added this line as this refactor impacts the work flow of how a simulation/program would be run with the module as people will now have to use the `ClassAEndDeviceLorawanMac` class instead of the `EndDeviceLorawanMac` class.